reports/_baseReport.js

```diff
@@ -4,7 +4,7 @@ function _BaseKeabbyReporter(baseConfig, reporter) {
 
   var krabbyReporter = function(config) {
     var args = Array.prototype.slice.call(arguments);
-    this.config = _.extend(baseConfig, config);
+    this.config = _.extend({}, baseConfig, config);
```

Comment:
Almost all of the tests fail without this. I know this is already in #20 so we can either merge that first and remove from this branch or close it out in favor of this one.